hack/quickstart-setup.sh

```diff
@@ -0,0 +1,322 @@
+#!/bin/bash
+
+#
+# Copyright 2022 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+export KIND_BIN=kind
+export YQ_BIN=yq
+
+source /dev/stdin <<< "$(curl -s https://raw.githubusercontent.com/Kuadrant/multicluster-gateway-controller/main/hack/.kindUtils)"
+source /dev/stdin <<< "$(curl -s https://raw.githubusercontent.com/Kuadrant/multicluster-gateway-controller/main/hack/.clusterUtils)"
+source /dev/stdin <<< "$(curl -s https://raw.githubusercontent.com/Kuadrant/multicluster-gateway-controller/main/hack/.cleanupUtils)"
+
+#TODO this needs to be update to the kuadrant git repo before merge
+MGC_REPO="github.com/laurafitzgerald/multicluster-gateway-controller.git"
+QUICK_START_HUB_KUSTOMIZATION=${MGC_REPO}/config/quickstarthub?ref=quickstart-script
+QUICK_START_SPOKE_KUSTOMIZATION=${MGC_REPO}/config/quickstartspoke?ref=quickstart-script
+ISTIO_KUSTOMIZATION=${MGC_REPO}/config/istio?ref=quickstart-script
+
+KIND_CLUSTER_PREFIX="mgc-"
+KIND_CLUSTER_CONTROL_PLANE="${KIND_CLUSTER_PREFIX}control-plane"
+KIND_CLUSTER_WORKLOAD="${KIND_CLUSTER_PREFIX}workload"
+
+set -e pipefail
+
+
+configureMetalLB () {
+  metalLBSubnet=${2}
+  echo "Creating MetalLB AddressPool"
+  cat <<EOF | kubectl apply -f -
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: example
+  namespace: metallb-system
+spec:
+  addresses:
+  - 172.32.${metalLBSubnet}.0/24
+---
+apiVersion: metallb.io/v1beta1
+kind: L2Advertisement
+metadata:
+  name: empty
+  namespace: metallb-system
+EOF
+}
+
+
+deployIstio() {
```

Comment:
I'd like to remove this function because the directory now has kustomization.yaml present but it's called twice. Once for the hub and once for the spoke. 
When moving it to the relevant quickstart kustomization i saw various errors.
For Hub I faced the error `error: resource mapping not found for name: "istiocontrolplane" namespace: "istio-system" from "STDIN": no matches for kind "IstioOperator" in version "install.istio.io/v1alpha1"
ensure CRDs are installed first` similar to the gateway class
And for spoke it's called from inside the ocm functions.

Reply:
To remove this you will need to remove the need for the `istioctl operator init` command below. That generates a load of stuff and applies it to the cluster. You can instead just check it in though and add it to the kustomization see https://github.com/mikenairn/multi-cluster-traffic-controller/tree/56059eb0fe82fcf99247aa2fffcd68f07aef331d/config/istio, only thing here you will hit the issue described here https://github.com/Kuadrant/multicluster-gateway-controller/pull/362/files#r1277269321

Reply:
```./bin/istioctl operator dump```

Reply:
@mikenairn i can't get to the comment that you're linking to above here. I checked in the changes from your branch to have kustomize install istio. The script is completing successfully. Is the issue you mentioned above still present? If so could you describe it more?